app/javascript/components/message.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { useSelector } from "react-redux";
+
+const Message = () => {
+  const greeting = useSelector((state) => state.message);
+     return ( 
+          <h1>{greeting.message}</h1>
+      );
+}
+ 
+export default Message;
```

Comment:
- Good job on the job done so far, but the frontend of your project does not seem to be rendering, Please fix this 🙏.
Your project:
![ethaethaeh](https://user-images.githubusercontent.com/73607512/230013697-e6db5425-2811-4edf-8757-c1d3d64b99df.PNG)

Reply:
@mckent05  i was trying to do with the ./bin/dev but i didnot past the database , can you help me with that ? please